api/src/resources/bucket.rs

```diff
@@ -168,3 +170,19 @@ impl Display for BucketType {
         }
     }
 }
+
+impl FromStr for FullName {
+    type Err = Error;
+    fn from_str(string: &str) -> Result<Self> {
+        let full_name_regex = Regex::new("^[^/]+/[^/]+$").unwrap();
```

Comment:
Same here, although, is there a way we can make the regex a constant so we don't have to modify it in two places? Also makes sure we don't have mismatching regexes.